lib/Dispute.php

```diff
@@ -64,17 +64,17 @@ class Dispute extends ApiResource
     const STATUS_WON = 'won';
 
     /**
+     * @param null|array $params
      * @param null|array|string $opts
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
      * @return \Stripe\Dispute the closed dispute
      */
-    // TODO: add $params to standardize signature
-    public function close($opts = null)
+    public function close($params = null, $opts = null)
     {
         $url = $this->instanceUrl() . '/close';
-        list($response, $opts) = $this->_request('post', $url, null, $opts);
+        list($response, $opts) = $this->_request('post', $url, $params, $opts);
```

Comment:
Curious if this is worth the breaking change versus just removing the method instead and requiring using services. That's a pretty subtle change for someone using that method to suddenly break because they pass an option as the first parameter.

Reply:
@dcr-stripe, @pakrym-stripe, @yejia-stripe, what do you think?

Reply:
Discussed as a team and we agreed to keep this.